apps/cms/.eslintrc

```
@@ -0,0 +1,3 @@
+{
+  "extends": "@sanity/eslint-config-studio"
```

```suggestion
  "extends": ["custom", "@sanity/eslint-config-studio"]
```
Her tror jeg du kan legge til våres egen `eslint` config, og extende begge.